mods/ts/rules/defaults.yaml

```diff
@@ -721,3 +721,28 @@
 	CustomSelectionSize:
 		CustomBounds: 220,220
 
+^Gate:
+	Inherits: ^Building
+	Valued:
+		Cost: 250
+	Health:
+		HP: 350
+	Armor:
+		Type: Heavy
+	LineBuildNode:
```

Comment:
This seems to be resolved by now:

![image](https://cloud.githubusercontent.com/assets/756669/12215089/c7790370-b6ad-11e5-8559-9e2ababb4cd6.png)